requirements.txt

```diff
@@ -33,3 +33,8 @@ django-sendfile
 django-polymorphic
 networkx
 rdflib>=4.1.0
+celery-with-redis
+django-celery
+scikit-learn
+nilearn
+pybraincompare
```

Comment:
I don't think those are in pypi so please add git link (see http://stackoverflow.com/questions/16584552/how-to-state-in-requirements-txt-a-direct-github-source)